### ISSUE TITLE
XSLT 7

### ISSUE THREAD

**Issue by mjb232** [ebeshero/DHClass-Hub#241]
So, Im drawing a blank: can one xpath down to a specific character within a text? I wanted to do some stuff with all the odd characters in the Dickinson poems (dashes, quotes, hyphens and the like).

**Comment by ebeshero** [comment]
@mjb232 Yes you can! :-) Use one of the string-functions: `contains()` or `matches()` to drill down into the text and retrieve and work with the punctuation marks. 

Check out the string-matching functions in [The XPath Functions We Use Most](http://dh.obdurodon.org/functions.xhtml) or read up in Michael Kay.

**Comment by mjb232** [comment]
duuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuh
